src/tests/test_utils/common_test_utils/src/test_helpers.cpp

```diff
@@ -0,0 +1,77 @@
+#include "common_test_utils/test_helpers.hpp"
+#include "openvino/op/tensor_iterator.hpp"
+
+#include "backend.hpp"
+
+bool ov::test::utils::is_tensor_iterator_exist(const std::shared_ptr<ov::Model>& func) {
+    const auto& ops = func->get_ops();
+    for (const auto& node : ops) {
+        const auto& ti = std::dynamic_pointer_cast<ov::op::v0::TensorIterator>(node);
+        if (ti) {
+            return true;
+        }
+    }
+    return false;
+}
+
+std::vector<ov::Tensor> ov::test::utils::interpret_model(const std::shared_ptr<ov::Model>& function,
+                                                         const std::map<std::shared_ptr<ov::Node>, ov::Tensor>& inputs) {
+    auto backend = ov::runtime::Backend::create();
```

Comment:
do we need to compute with interpreter backend?
maybe we can use template plugin directly?

It will also allow us to replace reference device (template plugin by default) and compare GPU versus CPU

Reply:
Will check

Reply:
Yes, we can do this, but I see no value here - it may be useful in `graph_comparator` only (in order to make some kind of cross_check_tool which was also deprecated and removed)

Reply:
Replaced with `infer_on_template()` in scope of another PR (already merged)